plugin/plugin.go

```diff
@@ -88,27 +88,31 @@ func (p *CLIPlugin) GetMetadata(ctx context.Context, req *proto.GetMetadataReque
 // DescribeKey returns the KeySpec of a key.
 func (p *CLIPlugin) DescribeKey(ctx context.Context, req *proto.DescribeKeyRequest) (*proto.DescribeKeyResponse, error) {
 	var resp proto.DescribeKeyResponse
+	req.ContractVersion = proto.ContractVersion
 	err := run(ctx, p.name, p.path, req, &resp)
 	return &resp, err
```

Comment:
Should we check the ContractVersion in the response before returning?

Reply:
There is no ContractVersion in the response according to our [spec](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#describe-key).

Reply:
How about other like `GenerateSignature`, `GenerateEnvelope`, `VerifySignature`?

Reply:
"supportedContractVersions" only exists in `get-plugin-metadata` response, so we can not check it for other commands.